source/CMakeLists.txt

```diff
@@ -21,8 +21,6 @@ ADD_DEFINITIONS("-DPREFIX=\"${PREFIX}\"")
 
 SET(PKG_MODULES
         libusbgx>=0.2.0
-	glib-2.0
```

Comment:
How about adding CMake option like BUILD_GADGETD_BACKEND and using it for conditional configuration and dependencies?

Reply:
Makes sense. pushed -f the next version.